src/setup/demo.md

```diff
@@ -0,0 +1,352 @@
+# Hands-on Substra demo
+
+Check this [page](https://www.substra.ai/en/demo) and get in touch with us, we will then provide you with instructions and credentials!
+
+## 0. Pre-requisites & Overview of the demo
+
+### Terminology
+
+To follow this tutorial for using the demo instance, it is assumed you have read the documentation of the Substra Framework and are familiar with its core concepts. In particular the following terms: organisation, asset, algo, objective, metric, opener, train task, test task.
+
+### Technical pre-requisites
+
+The demo instance is deployed on a cloud infrastructure. To use it you will need:
+
+- the credentials that were provided to you after you contacted us for accessing the demo instance
+- access to a computer (e.g. a laptop, a VM...) with an internet connection, on which you have sudo privileges. Preferably not a Windows machine, although this is still manageable (see below).
+
+### Overview: what you can do with the demo instance
+
+Although this can be inferred from the below tutorial by experienced readers, it might be good to be explicit about the proposed demo:
+
+- Substra Framework is meant to be instantiated as a distributed application among multiple partners. However, for this simple demo to be be accessible without any deployment operation and allow you to focus on the data scientist perspective, it is installed as a single instance on a cloud VM, in which 2 `organisations` have been created and live independently.
+- The credentials provided to you grant you access to only one of these 2 `organisations`. You will not be able to access or tamper with any asset of the second one; you can consider it as another company to which you don't have access.
```

Comment:
You have only access to one organization? so you run everything on one organization if you want to add your asset? is it relevant?

Reply:
No, 2 orgs are instanciated and you can use the second as well

Reply:
According to what people want to do, we can create credentials to both orgs